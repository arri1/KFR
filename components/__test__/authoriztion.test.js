import React from 'react';
import renderer from 'react-test-renderer';
import Authorization from '../routers/authorization';

test('renders correctly', () => {
  const tree = renderer.create(<Authorization/>).toJSON();
  expect(tree).toMatchSnapshot();
});