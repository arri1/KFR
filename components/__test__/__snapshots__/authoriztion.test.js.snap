// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#B8B8B8",
        "height": 50,
        "justifyContent": "center",
        "margin": 24,
        "opacity": 1,
        "width": 200,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 15,
          "fontWeight": "bold",
        }
      }
    >
      Sign up
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#B8B8B8",
        "height": 50,
        "justifyContent": "center",
        "margin": 24,
        "opacity": 1,
        "width": 200,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 15,
          "fontWeight": "bold",
        }
      }
    >
      Sign in
    </Text>
  </View>
</View>
`;
